core-service/src/main/java/com/crmfoodestablishment/coreservice/controller/MenuController.java

```diff
@@ -1,42 +1,56 @@
 package com.crmfoodestablishment.coreservice.controller;
 
 import com.crmfoodestablishment.coreservice.dto.MenuDto;
-import com.crmfoodestablishment.coreservice.service.MenuService;
+import com.crmfoodestablishment.coreservice.service.MenuServiceImpl;
 import jakarta.validation.Valid;
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import java.util.List;
 import java.util.UUID;
 
 @RestController
-@RequestMapping("api/v1/menu")
+@RequestMapping("api/menu")
 @RequiredArgsConstructor
 public class MenuController {
 
-    private final MenuService menuService;
+    private final MenuServiceImpl menuServiceImpl;
 
     @PostMapping
-    public UUID createMenu(@Valid @RequestBody MenuDto menuDto) {
-        return menuService.addMenu(menuDto);
+    public ResponseEntity<UUID> createMenu(@Valid @RequestBody MenuDto menuDto) {
+        UUID uuid = menuServiceImpl.addMenu(menuDto);
+        return new ResponseEntity<>(uuid, HttpStatus.CREATED);
     }
 
     @GetMapping
-    public List<MenuDto> getAllMenuList() {
-        return menuService.findAllMenu();
+    public ResponseEntity<List<MenuDto>> getAllMenuList() {
+        List<MenuDto> allMenuDtos = menuServiceImpl.findAllMenu();
+        return new ResponseEntity<>(allMenuDtos, HttpStatus.OK);
     }
 
     @GetMapping("/{id}")
-    public MenuDto getMenu(@PathVariable(name = "id") UUID uuid) {
-        return menuService.findByMenuUuid(uuid);
+    public ResponseEntity<MenuDto> getMenu(@PathVariable(name = "id") UUID uuid) {
```

Comment:
I think there is better to use @UUID validation annotation with String type on uuid parameter. So validation exception will be thrown not sterilization
Same with uuid parameters in methods below

Reply:
Let's discuss it at the meting